de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/IBPartnerBL.java

```diff
@@ -206,6 +206,10 @@
 		@Default
 		@NonNull
 		Comparator<User> comparator = Comparator.comparing(User::getName);
+
+		boolean forceActive;
+
+		boolean forceType;
```

Comment:
i am not sure this is a intention revealing name.
For me is not clear what's it's purpose.

Reply:
Added 
public enum IfNotFound
		{
			RETURN_DEFAULT_CONTACT, RETURN_NULL
		};